rpm-build/SPECS/securedrop-updater.spec

```diff
@@ -54,20 +54,27 @@ install -m 755 -d %{buildroot}/%{_bindir}
 install -m 755 -d %{buildroot}/%{_datadir}/applications/
 install -m 755 -d %{buildroot}/%{_datadir}/icons/hicolor/128x128/apps/
 install -m 755 -d %{buildroot}/%{_datadir}/icons/hicolor/scalable/apps/
+install -m 755 -d %{buildroot}/%{_libexecdir}/%{name}/migrations/
 install -m 755 -d %{buildroot}/%{_sharedstatedir}/%{name}/
 install -m 644 files/press.freedom.SecureDropUpdater.desktop %{buildroot}/%{_datadir}/applications/
 install -m 644 files/securedrop-128x128.png %{buildroot}/%{_datadir}/icons/hicolor/128x128/apps/securedrop.png
 install -m 644 files/securedrop-scalable.svg %{buildroot}/%{_datadir}/icons/hicolor/scalable/apps/securedrop.svg
 install -m 755 files/sdw-updater %{buildroot}/%{_bindir}/
 install -m 755 files/sdw-notify %{buildroot}/%{_bindir}/
 install -m 755 files/sdw-login %{buildroot}/%{_bindir}/
+install -m 755 files/migrations.py %{buildroot}/%{_libexecdir}/%{name}/
+install -m 644 files/migration_steps.py %{buildroot}/%{_libexecdir}/%{name}/
+install -m 644 migrations/* %{buildroot}/%{_libexecdir}/%{name}/migrations/
```

Comment:
This will install the new README file you wrote, right? Maybe that should be moved to a separate docs/ folder or something so it doesn't get packaged.

Reply:
Thanks for the catch, that is totally an oversight due to the late addition of `README.md`!